src/Microsoft.DotNet.CodeAnalysis.net45/Microsoft.DotNet.CodeAnalysis.net45.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Make sure you test this for internal builds, since I suspect that it needs to be v4.5 or it will break.